Cargo.toml

```diff
@@ -3,5 +3,28 @@ members = [
     "rust/crd", "rust/operator-binary"
```

Comment:
We could shorten this to `"rust/*"`

Reply:
This will break `cargo-version.py` release script. Can be fixed, no bid deal, just FYI.

Reply:
@razvan I would leave the final decision to you

Reply:
I would merge this and replace `cargo-version.py` with [cargo set-version](https://lib.rs/crates/cargo-edit) in the release scripts. We already use it in the GH actions.